src/components/TxData.js

```diff
@@ -369,7 +373,8 @@ class TxData extends React.Component {
       tokenClicked: token,
       unregisteredLoading: true,
     }, async () => {
-      const tokenDetails = await this.props.wallet.getTokenDetails(token.uid);
+      const wallet = getGlobalWallet();
+      const tokenDetails = await wallet.getTokenDetails(token.uid);
```

Comment:
```suggestion
      const { getTokenDetails } = getGlobalWallet();
      const tokenDetails = await getTokenDetails(token.uid);
```

Reply:
It's not possible to use this syntax with a HathorWallet method: it crashes the application.